spec/factories/product_price.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :product_price do
+    sequence(:category)   { |n| (n-1)%3 }
+    sequence(:price)    { |n| n }
+    association :product, title: "diaper", id: 1
+  end
```

Comment:
specify correct params for product_price

Reply:
How to define correct params?

Reply:
you should create them obvious and they should be understandable

Reply:
https://semaphoreci.com/community/tutorials/working-effectively-with-data-factories-using-factorygirl

Reply:
What about now?